src/main/java/Kaji.java

```diff
@@ -32,6 +33,9 @@ public Kaji(String filePath) {
     public void run() {
         ui.showWelcome();
         ui.showHelpList();
+        storage.createHistoryDir();
+        LocalDate date = java.time.LocalDate.now();
+        storage.createHistory(ui, date.toString());
```

Comment:
Maybe you call createHistoryDir in the method createAdmin in storage.
And for createHistory, you can call it in the method saveHistory